tor/core/validation.py

```diff
@@ -39,13 +67,17 @@ def verified_posted_transcript(post, config):
     # get source link, check all comments, look for root level comment
     # by the author of the post and verify that the key is in their post.
     # Return True if found, False if not.
-    linked_resource = config.r.submission(top_parent.id_from_url(top_parent.url))
+    linked_resource = config.r.submission(
+        top_parent.id_from_url(top_parent.url)
+    )
     # get rid of the "See More Comments" crap
     linked_resource.comments.replace_more(limit=0)
     for top_level_comment in linked_resource.comments.list():
-        if _author_check(post, top_level_comment) and _header_check(top_level_comment, config):
+        if (
+            _author_check(post, top_level_comment) and
+            _header_check(top_level_comment, config)
+        ):
             return True
-    # Did it get removed? Check their history.
-    # if _author_history_check:
-    #     return True
-    # return False
+
+    # Did their transcript get flagged by the spam filter? Check their history.
+    return _author_history_check(post, config)
```

Comment:
Is there any reason we wouldn't want to solely rely on this history check for the one source of truth for the transcription comment? Looping through the top-level comments seem like extra API calls that aren't needed if the author history would cover it on its own.

Reply:
Honestly, if a comment is removed, that's the target subreddit's fault not ours. We only care about providing the option of a transcription and giving credit to the author of it. Maybe it means we update the OP on /r/tor with a direct link to the transcription comment. I'm not sure.

Reply:
In the greater sense it's the Reddit spam filter that's the cause, but yes, I suppose that's valid. Mostly we've just been counting to see how active the spam filter is / if we need to intervene again with the admins. My worry is that if we don't check for removed comments we'll never know if the spam filter starts removing all of the volunteer posts or not, thus rendering our work useless.

Reply:
Ah, so we do want to have some validation tripwire, even if it doesn't catch all of the anti-spam measures. That way we can make sure it has a chance to be useful to others. I'd agree with keeping it as is, in that case.

Reply:
Can you elaborate on "validation tripwire"?

Reply:
By that I mean it's a tripwire, or early warning system, where we can tell something is either currently, or will very soon go wrong. In this case it's something that won't catch all of the times a transcriber's post is removed, but it will catch some of them and that's better than nothing.

The "validation" part is just pertaining to where the tripwire lives.